tidoop-mr-lib-api/src/mysql_driver.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-tidoop (FI-WARE project).
+ *
+ * fiware-tidoop is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-tidoop is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-tidoop. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with
+ * francisco dot romerobueno at telefonica dot com
+ */
+
+/**
+ * MySQL driver.
+ *
+ * Author: frb
+ */
+
+// module dependencies
+var mysql = require('mysql');
+var host = require('../conf/tidoop-mr-lib-api.json').mysql.host;
+var port = require('../conf/tidoop-mr-lib-api.json').mysql.port;
+var user = require('../conf/tidoop-mr-lib-api.json').mysql.user;
```

Comment:
Instead of requiring the same file over and over again, you should require it once and reuse that object. require() loads (it uses a cache) and compiles the file each time it is invoked... ;)